mkosi/__init__.py

```diff
@@ -4530,17 +4530,21 @@ def run_verb(args: Args, images: Sequence[Config], *, resources: Path) -> None:
         if args.verb == Verb.build and not args.force:
             check_outputs(config)
 
+    last = images[-1]
```

Comment:
Since I can't comment below: `last = image[-1]` happens again directly after this block and I think it would be helpful to add a comment there, that it's still needed, since `last` and `image[-1]` will not be the same anymore after line 4570.